ui/src/app/views/landing/vsphere-tkgs-wizard/env-details-step/env-details.component.ts

```diff
@@ -249,7 +249,7 @@ export class EnvDetailsComponent extends StepFormDirective implements OnInit {
             } else {
                 this.connected = false;
                 this.loadingState = ClrLoadingState.DEFAULT;
-                this.errorNotification = 'Some Error Occurred while Retrieving SSL Thumbprint';
+                this.errorNotification = 'An Error Occurred while Retrieving SSL Thumbprint ' + err.msg;
```

Comment:
You're correct - this should be rejected.  At this step, we don't have access to the data object since it's null or false.  Would the apiClient object be able to return its error description via this.apiClient.errorMessage in this case?

Reply:
this.errorNotification -> this object is globally maintained and will be used for logging. The only thing is we cannot use 'err' here. So lets revert this.